### DISCUSSION TITLE
Is it not strange to not support Turkey using SMS?

### DISCUSSION THREAD

**Discussion by selimq** [community/community#86997] [Other Feature Feedback, Questions, & Ideas]
I can not get authentication codes by SMS on my mobile phone from **Turkey**. There are 100+more country in list but no Turkey. What's the purpose/intent of this? Is it make sense to have strange countries even does anybody know but no Turkey?

https://docs.github.com/articles/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication

**Comment by faridnec** [comment]
Maalesef, mail ile yapabilirsiniz Authentication için yada 2FA Authenticator code kullanabilirsiniz. Ben öyle yaptım
